docs/kubernetes/how-to-deploy-jenkins-x-in-linode-kubernetes-engine/index.md

```diff
@@ -0,0 +1,365 @@
+lke7189-9006-5f051460a1e2   Ready    <none>   8h    v1.17.3
+lke7189-9006-5f0514617a87   Ready    <none>   8h    v1.17.3
+{{</ output>}}
+
+## Install `jx`
+
+To install Jenkins X, you need the command-line tool `jx`.
+
+You can find the instruction on [how to install `jx` on the official documentation.](https://jenkins-x.io/docs/install-setup/install-binary/)
+
+You can verify that the installation is successful with:
+
+```bash
+jx version
+```
+
+The output should be similar to:
+
+{{< output >}}
+Version        2.1.80
+Commit         b2bd447
+Build date     2020-06-23T15:34:17Z
+Go version     1.13.8
+Git tree state clean
+{{</ output>}}
+
+## Bootstrap Jenkins X
+
+Create a new folder and execute the `jx boot` command to start a new project:
+
+```bash
+mkdir jenkins-x
+cd jenkins-x
+jx boot
+```
+
+The command prompts the following message:
+
+{{< output >}}
+Creating boot config with defaults, as not in an existing boot directory with a git repository.
+No Jenkins X pipeline file jenkins-x.yml or no jx boot requirements file jx-requirements.yml found. You are not running this command from inside a Jenkins X Boot git clone
+To continue we will clone https://github.com/jenkins-x/jenkins-x-boot-config.git @ master to jenkins-x-boot-config
+? Do you want to clone the Jenkins X Boot Git repository? [? for help] (Y/n)
+{{</ output>}}
+
+_Yes_ is the default answer, so you can press enter and continue to the next step (spoiler, the next step fails).
+
+In the next step, the CLI clones the Jenkins X configuration repo and validates your configuration.
+
+The command-line tool expects the cluster name to be in the format `<project id>_<zone>_<cluster name>` — which is the default if you want to use Google Kubernetes Engine.
+
+Since Linode Kubernetes Engine doesn't follow the same naming pattern, the validation fails.
+
+{{< output >}}
+Cloning https://github.com/jenkins-x/jenkins-x-boot-config.git @ master to jenkins-x-boot-config
+Attempting to resolve version for boot config https://github.com/jenkins-x/jenkins-x-boot-config.git from https://github.com/jenkins-x/jenkins-x-versions.git
+Booting Jenkins X
+
+STEP: validate-git command: /bin/sh -c jx step git validate in dir: /tmp/jenkinsx/jenkins-x-boot-config/env
+
+STEP: verify-preinstall command: /bin/sh -c jx step verify preinstall --provider-values-dir="kubeProviders" in dir: /tmp/jenkinsx/jenkins-x-boot-config
+
+error: : unable to parse lke7389 as <project id>_<zone>_<cluster name>
+error: failed to interpret pipeline file /tmp/jenkinsx/jenkins-x-boot-config/jenkins-x.yml: failed to run '/bin/sh -c jx step verify preinstall --provider-values-dir="kubeProviders"' command in directory '/tmp/jenkinsx/jenkins-x-boot-config', output: ''
+{{</ output>}}
+
+But don't worry, this error can be fixed.
+
+If you inspect the current folder, you should notice that `jx` created a `jenkins-x-boot-config` folder.
+
+In that folder, there's a `jx-requirement.yaml` file with the details of your cluster.
```

Comment:
jx-requirements.yml is the file that needs to be edited.
<img width="536" alt="image" src="https://user-images.githubusercontent.com/37059749/88457278-bed6e680-cea2-11ea-8314-3e7d2af78bee.png">

Reply:
Fixed